package.json

```
@@ -3,20 +3,28 @@
   "name": "@locomotivemtl/boilerplate",
   "title": "Locomotive Boilerplate",
   "version": "1.0.0",
+  "engines": { "node": "???" },
```

```suggestion
  "engines": {
    "node": "^14.15"
  },
```

This should reflect the version defined in the `.nvmrc`.